server/api/lib/tags/light_rail.js

```diff
@@ -0,0 +1,32 @@
+const TAG_NAME = 'רכבת קלה';
+
+const doesTagApply = async (plan, tagsResources) => {
+    const getReturnDict = (stringInPlanName) => {
+      return {
+          plan_id: plan.id,
+          tag_id: tagsResources.tagNameToTagId[TAG_NAME],
+          display_score: 0, /* TODO: Add the correct display score here */
+          created_by_data_rules: JSON.stringify([{ rule: `has "${stringInPlanName}" in plan name` }])
+      };
+    };
+
+    const planName = plan.attributes.PL_NAME;
```

Comment:
Do all plans have attributes?

Reply:
They should, at least when they come from the DB. (Bookshelf should handle this. If he can't bring attributes, he should fail in the db fetch phase)

Reply:
ooh ok